Docs/source/unreal-pages/get-started.md

```diff
@@ -23,6 +23,14 @@ To add the AltTester® Unreal SDK to your Unreal application, you first need to
 
 ```
 
+```eval_rst
+.. important::
+
+    To download the AltTester® Plugin on **macOS**, you can use the **wget** command for convenience. Otherwise, if downloading via a browser, ensure to either Allow the files from *Privacy & Security Settings* if macOS prevents the file from being opened or **remove the quarantine attribute** from the downloaded files. 
```

Comment:
"...ensure to either Allow the files..."here I'm not sure if "Allow" should be with lower case or is the text from the button and then should be marked with stars as the other file/command names.